packages/backend/api/src/models/follow-model.ts

```diff
@@ -0,0 +1,28 @@
+import { db } from '@app/backend-shared';
+
+async function getFollowersCount(userId: number): Promise<number> {
+  const result = await db
+    .selectFrom('follow_up as f')
+    .innerJoin('user as u', 'u.id', 'f.follower_id')
+    .select((eb) => eb.fn.countAll().as('count'))
+    .where('f.followee_id', '=', userId)
+    .executeTakeFirst();
+
+  return Number(result?.count ?? 0);
```

Comment:
Si tu envoies un générique ici, tu n'as plus besoin de `Number()`
```suggestion
  const result = await db
    .selectFrom('follow_up as f')
    .innerJoin('user as u', 'u.id', 'f.follower_id')
    .select((eb) => eb.fn.countAll<number>().as('count'))
    .where('f.followee_id', '=', userId)
    .executeTakeFirst();

  return result?.count ?? 0
```